### ISSUE TITLE
Installation aborted

### ISSUE THREAD

**Issue by GayatriVar** [intel/haxm#437]
**Describe the Bug**

Summary: 
While installing HAXM it says that the system requirements does not match and the installation has aborted.
I have tried enabling virtualization in BIOS , also tried disabling Hyper-V , I am still facing the same problem. Please help me to solve my problem.
![1](https://user-images.githubusercontent.com/98007156/150069749-9cc9bbe8-1d78-4316-b3ad-3e6df0ee7400.JPG)
![2](https://user-images.githubusercontent.com/98007156/150069755-5e0fa262-9b37-4dc7-a751-51afc5256e89.JPG)
![3](https://user-images.githubusercontent.com/98007156/150069758-0a558cb2-550f-4f7e-8ead-f1816e728c31.JPG)

**Host Environment**

* HAXM version: 

* Host OS version: 

* Host OS architecture: x86\_64

* Host CPU model: 

* Host RAM size: 

* (Optional) Host computer model: 

**Guest Environment**

* Android Emulator or QEMU version: 

* Guest OS version: 

* Guest OS architecture: 

**To Reproduce**

1. 
2. 
3. 

**Expected Behavior**

Expectation: 

**Reproducibility**

**Diagnostic Information**

Host crash dump: 

HAXM log: 

Android Emulator or QEMU log: 

Screenshots: 

**Additional context**

None

**Comment by wcwang** [comment]
Thanks for your report. Currently HAXM only supports Intel CPU. From your installation log, you were trying to install HAXM onto AMD CPU. Thanks.

**Comment by GayatriVar** [comment]
Thank you . So what can I do for a successful installation?

**Comment by wcwang** [comment]
You cannot install HAXM on this AMD CPU based machine. If you switch to another PC running on Intel CPU, you can successfully install HAXM. Thanks.

**Comment by GayatriVar** [comment]
So you mean that I cannot install this on my laptop? if that is the case
can I install HAXM in any other way?

**Comment by wcwang** [comment]
No, you cannot install HAXM on this laptop in any approach as HAXM does not support AMD CPU. Thanks.

**Comment by gonzaloV999** [comment]
I`ve the same problem, but on my laptop a I have an INTEL I5 6th CPU. so I cant understand why this happend

**Comment by wcwang** [comment]
Thanks for your report. Could you find the installation log %TEMP%\haxm_install-*.log and check your host environment according to the [wiki page](https://github.com/intel/haxm/wiki/Windows-Installation-Environment-Check-and-Configurations)?

**Comment by GayatriVar** [comment]
My problem got resolved. Thank you.

**Comment by pantchayan** [comment]
Same issue with following config

![image](https://user-images.githubusercontent.com/49815429/151341865-9a71135d-671b-4572-8690-948d9d2637fd.png)

**Comment by wcwang** [comment]
Thanks for your report. Have you tried the previous comments? If not, could you find the installation log %TEMP%\haxm_install-*.log and check your host environment according to the [wiki page](https://github.com/intel/haxm/wiki/Windows-Installation-Environment-Check-and-Configurations) first?

**Comment by pantchayan** [comment]
Thanks! 
Had to manually turn on Virtualisation through BIOS.

![sasa](https://user-images.githubusercontent.com/49815429/151347364-ccba7b39-5a80-417d-90ab-6fd30724a805.PNG)

**Comment by wayne-ma** [comment]
@pantchayan ,yes, enable VT through BIOS is a must for HAXM.
